doc/source/whatsnew/v1.1.5.rst

```diff
@@ -29,6 +29,7 @@ Fixed regressions
 - Fixed performance regression in ``df.groupby(..).rolling(..)`` (:issue:`38038`)
 - Fixed regression in :meth:`MultiIndex.intersection` returning duplicates when at least one of the indexes had duplicates (:issue:`36915`)
 - Fixed regression in :meth:`.GroupBy.first` and :meth:`.GroupBy.last` where ``None`` was considered a non-NA value (:issue:`38286`)
+- Fixed regression in :func:`read_sql_table` raising a ``sqlalchemy.exc.OperationalError`` when column names contained a percentage sign (:issue:`37517`)
```

Comment:
move to 1.2 i think we could put this in.

Reply:
Moved it to 1.2 and as bug fix, since it's not a regression fix if I understood you correctly.